playbooks/allocate_pubcloud.yml

```diff
@@ -15,6 +16,12 @@
           count: "{{ count }}"
           key_name: "jenkins"
           region: "{{ region }}"
+          # TODO(odyssey4me):
+          # We no longer need to use a lookup here once
+          # we shift to using Ansible 2.3+. In Ansible
+          # 2.2 only a string works here, not a file
+          # path.
```

Comment:
This comment should make it clear that its a problem with the rax module, not ansible lookups.

Eg: In Ansible<2.3 there is a bug which prevents the rax module from reading user_data files. To work around that bug, we supply the contents of the user_data file rather than the path to it.  This bug is fixed in 2.3, so once we upgrade to 2.3+, we can set `user_data: user_data_path`

Reply:
Done